natlas-server/config.py

```diff
@@ -79,3 +79,10 @@ class Config(object):
     opencensus_enable = casted_bool(os.environ.get("OPENCENSUS_ENABLE", False))
     opencensus_sample_rate = float(os.environ.get("OPENCENSUS_SAMPLE_RATE", 1))
     opencensus_agent = os.environ.get("OPENCENSUS_AGENT", "127.0.0.1:55678")
+
+
+class TestConfig(Config):
```

Comment:
Why is this in the production code? Is it possible to move this to the tests/ folder?

Reply:
Idk what you're talking about, it's clearly in it's own tests/config.py file 🤠